soapboxpb/environment.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+package soapbox;
+option go_package = "soapboxpb";
+
+import "application.proto";
+
+service Environments {
+    rpc ListEnvironments (ListEnvironmentRequest) returns (ListEnvironmentResponse) {}
+    rpc GetEnvironment (GetEnvironmentRequest) returns (GetEnvironmentResponse) {}
+    rpc CreateEnvironment (CreateEnvironmentRequest) returns (CreateEnvironmentResponse) {}
+    rpc DestroyEnvironment (DestroyEnvironmentRequest) returns (DestroyEnvironmentResponse) {}
+    rpc CopyEnvironment (CopyEnvironmentRequest) returns (CopyEnvironmentResponse) {}
+}
+
+message ListEnvironmentRequest {}
+
+message EnvironmentSummary {
+    int32 id = 1;
+    string name = 2;
+}
+
+message ListEnvironmentResponse {
+    repeated EnvironmentSummary environments = 1;
```

Comment:
Would that return all of the environment variables too? If so, that seems like it might be a lot to include.